addons/mail/static/src/new/rtc/call_main.js

```diff
@@ -101,7 +101,7 @@ export class CallMain extends Component {
             return;
         }
         const { width, height } = this.grid.el.getBoundingClientRect();
-        const aspectRatio = 16 / 9;
+        const aspectRatio = this.props.minimized ? 1 : 16 / 9;
```

Comment:
I'm not seeing what this does, I never have a 1:1 ratio unless I don't know how to reach that case.